### DISCUSSION TITLE
打开兑换一段时间会自动退出

### DISCUSSION THREAD

**Discussion by YuziYYDS** [Ljzd-PRO/Mys_Goods_Tool#148] [Q&A]
进入兑换模式后一段时间就自己闪退了其他时间没有问题就在兑换模式会这样？有人知道是怎办回事吗？求解    急~

**Comment by YuziYYDS** [comment]
![Snipaste_2023-07-26_08-29-32](https://github.com/Ljzd-PRO/Mys_Goods_Tool/assets/135944633/b680001a-0842-4d88-b457-69cbc28d183c)

**Reply by YuziYYDS** [reply]
点确定后一段时间后就闪退了

**Reply by kuronekoda** [reply]
能兑换到东西吗，我现在啥都抢不了，返回1028

**Reply by YuziYYDS** [reply]
从来没在2.几抢到过东西

**Comment by Aellyt** [comment]
提前2分钟左右开，太早确实这样

**Reply by YuziYYDS** [reply]
ok
